api/package.json

```diff
@@ -6,10 +6,12 @@
   "scripts": {
     "start": "node app",
     "test": "node_modules/.bin/jest --verbose",
-    "lint": "eslint . --no-fix --ignore-pattern 'node_modules' --ext .js",
-    "lint-fix:1": "eslint --fix --ignore-pattern 'node_modules' --ext .js",
-    "lint-fix:2": "prettier ./**/*.js --write --loglevel warn",
-    "lint-fix": "npm-run-all -l -s -c lint-fix:*"
+    "lint": "npm-run-all -l -s -c lint:*",
+    "lint:1": "eslint ./**/*.js",
```

Comment:
npm run lint:1 is failing

Reply:
The command is failing, or the linter is reporting lint errors?

Reply:
```npm run lint``` in the api directory, you'll see the error.

Reply:
Locally, the command is working for me, and it reports a bunch of linting errors.  I didnt fix them as part of not linting code that doesnt have changes in it though.

Reply:
Im not quite sure what error you're seeing with the linting.  For me locally, the command runs, reports linting errors, and ultimately reports that it failed, but only because of the linting issues.  The command itself executed fully, as intended.

Are you seeing an actual error that prevents the lint commands from running at all?

Reply:
![image](https://user-images.githubusercontent.com/51633/72553876-72a14e80-384e-11ea-9064-741cc827b461.png)

Reply:
![image](https://user-images.githubusercontent.com/51633/72553985-ae3c1880-384e-11ea-9fe3-b4c1d7b33af4.png)

Reply:
Hmm, not sure what thats about.

The pattern error makes no sense to me.  Why would it have anything to do with node_modules, let alone as the pattern.  node_modules is also in the ignore file, as usual.

Reply:
I wonder if it has to do with a difference in shell or OS?

Reply:
Is there a version of that command that works for you locally? Maybe you need brackets around the pattern?

If you change it to:

```
"lint:1": "eslint \"./**/*.js\"",
```

does it work for you?

Reply:
Any luck?

Reply:
In the screenshot it shows that you're running the linter against the node_modules directory.  Even though the eslintignore is there, it will catch any directory that has a .js in it's name.

Reply:
I can't seem to reproduce this issue, so I'm not sure how to resolve this.  

The .eslintignore seems to be working correctly for me, as when I add '--no-ignore' it tries to lint the migration files that it would normally skip.  Even with the ignore commented out entirely, it still doesn't try to lint node_modules, because I think eslint ignores it by default.